package.json

```diff
@@ -161,7 +161,7 @@
     "@csstools/css-parser-algorithms": "^3.0.2",
     "@csstools/css-tokenizer": "^3.0.2",
     "@csstools/media-query-list-parser": "^4.0.0",
-    "@csstools/selector-specificity": "^4.0.0",
+    "@csstools/selector-specificity": "^5.0.0",
```

Comment:
This only matches the update to `postcss-selector-parser`: https://github.com/csstools/postcss-plugins/blob/main/packages/selector-specificity/CHANGELOG.md#500